app/views/categories/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<header class="d-flex justify-content-between align-items-center header">
+  <%= link_to image_tag("https://img.icons8.com/officel/344/back.png"),categories_path, class: "back_btn" %>
+  <h5>Transactions</h5>
```

Comment:
- [ ] When the user creates a new category, after clicking save it should send him to the homepage, instead in your app it sends to the category index page

Reply:
Well, it is programmed as such since the homepage is supposed to be set to the category page as directed in the program requirement, https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Home%20page%20(catagories,a%20new%20category.